blog/posts/_data.json

```diff
@@ -0,0 +1,20 @@
+{
+  "post-2": {
+    "title": "Sample Post 2",
+    "date": "January 4, 2015",
```

Comment:
For date stamps we should use ISO format `yyyy-mm-dd` and use a library such as [moment](http://momentjs.com/) to format them however we please. 

I recognize that this is dummy data, so I'll capture this in the [documentation](https://github.com/GothamSass/chrysler/wiki/Contributing-to-the-blog)